cmd/gaiad/cmd/root.go

```diff
@@ -82,6 +82,7 @@ func initAppConfig() (string, interface{}) {
 	srvCfg := serverconfig.DefaultConfig()
 	srvCfg.StateSync.SnapshotInterval = 1000
 	srvCfg.StateSync.SnapshotKeepRecent = 10
+	srvCfg.BaseConfig.IAVLDisableFastNode = false // enable fastnode by default
```

Comment:
Hi, @faddat, can you describe a bit more about the  `IAVLDisableFastNode ` ? what is the benefit and concerns if we set it `true` or `false`.  Is there some docs i can check? Thanks!

Reply:
If we set it to false here, then we should set it to false in v8.  I recommend false.

Reply:
false means we will use fast node.  I recommend using it.

Reply:
`fast node` means [fast sync](https://docs.tendermint.com/v0.34/tendermint-core/fast-sync.html) ? Do you know why the fafault is set to be true ? If we change it, what would be the impact on a node ?

Reply:
no, it refers to iavl fast node